src/status_im/ui2/screens/chat/photo_selector/view.cljs

```diff
@@ -0,0 +1,107 @@
+(ns status-im.ui2.screens.chat.photo-selector.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo.components.safe-area :as safe-area]
+            [quo2.core :as quo2]
+            [i18n.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.components.notifications.info-count :as info-count]
+            [react-native.linear-gradient :as linear-gradient]
+            [status-im.ui2.screens.chat.photo-selector.style :as style]))
+
+(def selected (reagent/atom []))
+
+(defn small-image [item]
+  [rn/view
+   [rn/image {:source {:uri item}
+              :style  {:width         56
+                       :height        56
+                       :border-radius 8
+                       :margin-bottom 20}}]
+   [rn/touchable-opacity
+    {:on-press (fn [] (reset! selected (vec (filter #(-> % (not= item)) @selected))))
+     :style    (style/remove-photo-container)}
+    [quo2/icon :i/close {:color colors/white :size 12}]]])
+
+(defn bottom-gradient []
+  [:f>
+   (fn []
+     (let [safe-area (safe-area/use-safe-area)]
+       (when (> (count @selected) 0)
+         [linear-gradient/linear-gradient
+          {:colors [:black :transparent]
+           :start  {:x 0 :y 1}
+           :end    {:x 0 :y 0}
+           :style  (style/gradient-container safe-area)}
+          [rn/flat-list {:key-fn                  (fn [item] item)
+                         :render-fn               small-image
+                         :data                    @selected
+                         :horizontal              true
+                         :content-container-style {:padding-horizontal 20 :margin-top 12}
+                         :separator               [rn/view {:style {:width 12}}]}]
+          [rn/view {:style (style/buttons-container safe-area)}
+           [quo2/button {:type     :grey
+                         :style    {:flex 0.48}
+                         :on-press #(js/alert "Add text: to be implemented")}
+            (i18n/label :t/add-text)]
+           [quo2/button {:style    {:flex 0.48}
+                         :before   :send
+                         :on-press #(do
+                                      (rf/dispatch [:chat.ui/send-current-message])
+                                      (reset! selected [])
+                                      (rf/dispatch [:bottom-sheet/hide]))}
```

Comment:
Can we make these two into one `dispatch` by `hide-sheet-and-dispatch`?

Reply:
`hide-sheet-and-dispatch` is a custom method used in files like actions drawer where it is called multiple times. 
Here it wouldn't add much value I think.